addons/html_editor/static/src/core/content_editable_plugin.js

```diff
@@ -15,7 +15,7 @@ export class ContentEditablePlugin extends Plugin {
     static id = "contentEditablePlugin";
     resources = {
         normalize_handlers: withSequence(5, this.normalize.bind(this)),
-        clean_for_save_handlers: this.cleanForSave.bind(this),
+        clean_for_save_handlers: withSequence(Infinity, this.cleanForSave.bind(this)),
```

Comment:
Maybe use `END` or `END + 1` by `import { END } from "@html_builder/utils/option_sequence";`, so it more aligns with the sequence range

Reply:
My bad, I thought the plugin was in the `html_builder`.I think we didn't have it in the `html_editor` for a system-wise sequence actually, we most of time were using the raw number.

In this case I think we should use `Infinity` as you did or `100` since we shouldn't introduce the dependency on `html_builder`. I checked in the code base and the maximum number range is 100 so far. For `clean_for_save_handlers`, I didn't find a case with a sequence, so either `Infinity` or `100` should be fine.

Since this manipulation of `contenteditable` should happen at the latest, `Infinity` might be a better solution :pray: